static/src/js/zustand/slices/__tests__/createSavedProjectSlice.test.ts

```diff
@@ -316,10 +318,11 @@ describe('createSavedProjectSlice', () => {
           }
         })
 
-        expect(actions.handleError).toHaveBeenCalledTimes(1)
-        expect(actions.handleError).toHaveBeenCalledWith({
+        const { errors } = useEdscStore.getState()
+        expect(errors.handleError).toHaveBeenCalledTimes(1)
+        expect(errors.handleError).toHaveBeenCalledWith({
           action: 'getProject',
-          error: 'Mock query error',
+          error: new Error('Mock query error'),
```

Comment:
same on this one just mainly curious why we need to instantiate it as Error in these?